proto/ethermint/feemarket/v1/feemarket.proto

```diff
@@ -13,8 +15,11 @@ message Params {
   // elasticity multiplier bounds the maximum gas limit an EIP-1559 block may
   // have.
   uint32 elasticity_multiplier = 3;
-  // initial base fee for EIP-1559 blocks.
-  int64 initial_base_fee = 4;
+  // base fee for EIP-1559 blocks.
+  string base_fee = 4 [
```

Comment:
you need to use `6` here and use `reserved` for the `initial_base_fee`.

https://github.com/tharsis/ethermint/blob/bd0f073704336700f6b3e8cdabaf2842216ab7ee/proto/ethermint/evm/v1/evm.proto#L102-L104

Reply:
cc: @danburck @yihuang @loredanacirstea @ramacarlucho @crypto-facs 

This is relevant in case you are modifying an existing field.

Reply:
it should be set accordingly to your network traffic and pricing strategy. 
`elasticity_multiplier`  defines how much the gas price can vary between blocks and `base_fee_change_denominator` defines your target gas used per block.